sql/src/test/java/io/crate/integrationtests/DDLIntegrationTest.java

```diff
@@ -191,6 +191,18 @@ public void testCreateGeoShapeExplicitIndex() throws Exception {
         assertEquals(expectedMapping, getIndexMapping("test"));
     }
 
+    @Test
+    public void testCreateColumnWithDefaultExpression() throws Exception {
+        execute("create table test (col1 text default 'foo')");
+        ensureYellow();
```

Comment:
```suggestion
        ensureYellow();
```
Shouldn't be needed to retrieve the index mapping. The `create table` only returns when the cluster state has been updated (shard initialization is async, but that's not required here)